src/sections/reviews.liquid

```diff
@@ -20,11 +20,9 @@
             <li class="reviews__list-item" data-review-id="{{ forloop.index }}">
               <header>
                 <span class="review__stars">
-                  {% for index in (1..5) %}
+                  {% for index in (1..review.rating) %}
                     <input type="radio" name"rating" value="{{ index }}"  />
```

Comment:
shouldn't this also have a tabindex of -1 if you don't want it to be tabbable?

Reply:
Niiiiiice catch, updated ;)